web/src/flow/stages/prompt/PromptStage.ts

```diff
@@ -149,7 +149,7 @@ export class PromptStage extends BaseStage<PromptChallenge, PromptChallengeRespo
                     ${prompt.choices
                         ?.map((choice) => {
                             return `<option
-                            value=${choice}
+                            value="${choice}"
                             ${prompt.placeholder === choice ? "selected" : ""}
```

Comment:
I must've missed this in the previous PR

```suggestion
                            ?selected=${prompt.placeholder === choice}
```